lib/logstash/filters/geoip.rb

```diff
@@ -144,11 +146,22 @@ def auto_target_from_source!
 
 
   def setup_filter(database_path)
+    @healthy_database = true
     @database = database_path
     @logger.info("Using geoip database", :path => @database)
     @geoipfilter = org.logstash.filters.geoip.GeoIPFilter.new(@source, @target, @fields, @database, @cache_size, ecs_compatibility.to_s)
   end
 
+  # call by DatabaseManager
+  def expire_action
+    fail_filter
+  end
+
+  def fail_filter
+    @logger.info("geoip plugin stop filtering")
```

Comment:
I suggest moving this to warn:
```suggestion
    @logger.warn("geoip plugin will stop filtering and will tag all events with the '_geoip_expired_database' tag.")
```

should we check that we are flipping `@health_database` from true to false? or can it happen that every day we're printing the message above ?

Reply:
I updated the log message to print the value of health_database explicitly. Nice suggestion